src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTag.java

```diff
@@ -6,30 +6,34 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
 
 /**
+ * Config:
+ * tag = &#64;author
+ * tagFormat = \\S
+ *
  * Testing tag writing
```

Comment:
is it possible to separate `Config` comments from actual example? it is a little bit confusing to understand :)

Reply:
Done.